ibis/backends/dask/execution/selection.py

```diff
@@ -117,13 +116,10 @@ def compute_projection_column_expr(
     )
 
     result = execute(expr, scope=scope, timecontext=timecontext, **kwargs)
+    result = util.coerce_to_output(result, expr, data.index)
```

Comment:
nbd, but should consistently import `coerce_to_output` (or just use `util`, prefer the first)